src/poprox_recommender/domain/article.py

```diff
@@ -1,12 +1,31 @@
 from datetime import datetime, timezone
+from typing import List, Optional
 from uuid import UUID
 
 from pydantic import BaseModel
 
 
+class Entity(BaseModel):
+    entity_id: Optional[UUID] = None
+    external_id: Optional[str] = None
+    name: str
+    entity_type: str
+    source: str
+    raw_data: str
+
+
+class Mention(BaseModel):
```

Comment:
I'm curious why `article_id` and `mention_id` are Optional here.

Reply:
These can exist in memory before they get inserted into the database and at that point we don't know the ids yet, because they're auto-assigned by the database on insert

Reply:
Gotcha. That makes sense